windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.adoc

```diff
@@ -14,12 +14,27 @@ You can create a Windows `MachineSet` object to serve a specific purpose in your
 * You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
 * You are using a supported Windows Server as the operating system image.
 +
-Use the following `aws` command to query valid AMI images:
+Use one of the the following `aws` commands, as appropriate for your Windows Server release, to query valid AMI images:
++
+.Example Windows Server 2022 command
++
+[source,terminal]
+----
+$ aws ec2 describe-images --region <aws_region_name> --filters "Name=name,Values=Windows_Server-2022*English*Core*Base*" "Name=is-public,Values=true" --query "reverse(sort_by(Images, &CreationDate))[*].{name: Name, id: ImageId}" --output table
+----
++
+.Example Windows Server 2019 command
 +
 [source,terminal]
 ----
-$ aws ec2 describe-images --region <aws region name> --filters "Name=name,Values=Windows_Server-2019*English*Full*Containers*" "Name=is-public,Values=true" --query "reverse(sort_by(Images, &CreationDate))[*].{name: Name, id: ImageId}" --output table
+$ aws ec2 describe-images --region <aws_region_name> --filters "Name=name,Values=Windows_Server-2019*English*Core*Base*" "Name=is-public,Values=true" --query "reverse(sort_by(Images, &CreationDate))[*].{name: Name, id: ImageId}" --output table
 ----
++
+--
+where:
+
+<aws_region_name>:: Specifies the name of your AWS region.
```

Comment:
Is there a link to these regions to add to the Additional resources? (Or is that something the admins need to have from AWS?)

Reply:
I've spot-checked this. It doesn't seem that we tell folks how to get the region. It is in the Web console.